application.go

```diff
@@ -361,7 +369,7 @@ func (r *Application) AddEnv(name, value string) *Application {
 	if r.Env == nil {
 		r.EmptyEnvs()
 	}
-	(*r.Env)[name] = value
+	(r.Env)[name] = value
```

Comment:
Ah I just understood this difference is because this PR changes the `Env` to not be a pointer anymore. Is this actually right? I think this has subtle differences in JSON marshalling and also it is an API break. So do we really have to do this?